test/unit_test/passes/onnx/test_quantization.py

```diff
@@ -52,6 +52,20 @@ def test_quantization(calibrate_method, tmp_path):
     assert out is not None
 
 
+def test_dynamic_quantization(tmp_path):
```

Comment:
do we need failure case?

Reply:
I am not sure what a failure case would look like since the default config and search points are valid. The bug cannot be reproduced for a failure case since it is not directly related to this pass or mode.

Reply:
so, I suppose the test case will catch the bug if the production code is reverted?

Reply:
yes, this test fails if I revert the param type fix.